packages/openactive-integration-tests/test/helpers/logger.js

```diff
@@ -1,3 +1,27 @@
+// TODO fix these issues!
```

Comment:
Might be a good issue to raise and do, for maintainability

Reply:
Good point @civsiv . I've created an issue to do that, and broadly fix all the ESLint sins, here: https://github.com/openactive/openactive-test-suite/issues/648